src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/sessiondays/SessionDaysReportTest.java

```diff
@@ -84,13 +79,13 @@ public void shouldNotShowCaseWithNoHearings() {
 
         var reportData = sessionDaysReport.generateReport(MANCHESTER_LISTING_CASE_TYPE_ID, DATE_FROM, DATE_TO);
         assertCommonValues(reportData);
-        assertEquals("0", reportData.getReportSummary().getFtSessionDaysTotal());
-        assertEquals("0", reportData.getReportSummary().getPtSessionDaysTotal());
-        assertEquals("0", reportData.getReportSummary().getOtherSessionDaysTotal());
-        assertEquals("0", reportData.getReportSummary().getSessionDaysTotal());
-        assertEquals("0", reportData.getReportSummary().getPtSessionDaysPerCent());
-        assertEquals(0, reportData.getReportSummary2List().size());
-        assertEquals(0, reportData.getReportDetails().size());
+        Assertions.assertEquals("0", reportData.getReportSummary().getFtSessionDaysTotal());
+        Assertions.assertEquals("0", reportData.getReportSummary().getPtSessionDaysTotal());
+        Assertions.assertEquals("0", reportData.getReportSummary().getOtherSessionDaysTotal());
+        Assertions.assertEquals("0", reportData.getReportSummary().getSessionDaysTotal());
+        Assertions.assertEquals("0", reportData.getReportSummary().getPtSessionDaysPerCent());
+        Assertions.assertEquals(0, reportData.getReportSummary2List().size());
```

Comment:
Small optional one from me would maybe just to import assertEquals - rather than having Assertions.assertEquals every time

Reply:
refactored